packages/frontend/src/components/MenuBar/DropDownMenu.tsx

```diff
@@ -3,25 +3,26 @@ import { useWindowSize } from "../../../utils/useWindowSize";
 import { Avatar } from "../Avatar";
 import styles from './menubar.module.css';
 import { Footer } from '..';
-import { ChevronRightIcon, CogIcon, InboxFillIcon, MenuGridIcon, PersonChatIcon } from '@navikt/aksel-icons';
+import { ChevronRightIcon, InboxFillIcon, MenuGridIcon, PersonChatIcon } from '@navikt/aksel-icons';
 import { Button } from '@digdir/designsystemet-react';
 import { Hr } from ".";
-import cx from 'classnames';
-import { useState } from "react";
-import { DropdownSubMenu, SubeMenuSelection } from "./DropDownSubMenu";
+import { DropdownSubMenu, SubMenuSelection } from "./DropDownSubMenu";
 import { DropDownMenuItem } from "./DropDownMenuItem";
 
 interface DropdownMenuProps {
   showDropdownMenu: boolean,
   name: string,
   companyName?: string
   onClose: () => void
+  showSubMenu: SubMenuSelection
+  setShowSubMenu: (showSubMenu: SubMenuSelection) => void
 }
 
-export const DropdownMenu: React.FC<DropdownMenuProps> = ({ showDropdownMenu, name, companyName, onClose }) => {
+export const DropdownMenu: React.FC<DropdownMenuProps> = ({ showDropdownMenu, name, companyName, onClose, showSubMenu, setShowSubMenu }) => {
   const { t } = useTranslation();
   const { isMobile } = useWindowSize();
-  const [showSubMenu, setShowSubMenu] = useState<SubeMenuSelection>('none')
+  const companyNameTitleCase = toTitleCase(companyName || '')
+  const nameTitleCase = toTitleCase(name || '')
```

Comment:
toTitleCase kan kanskje tillate undefined, så slipper du å tvinge gjennom tom streng som string for hver gang?